docker-storage-setup.sh

```diff
@@ -848,6 +848,17 @@ reset_storage() {
     rm -f $DOCKER_STORAGE
 }
 
+# Config dumping
+
+dump_config() {
+    echo STORAGE_DRIVER=$STORAGE_DRIVER
+    echo VG=$VG
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+        echo LV=$POOL_LV_NAME
+    fi
+    echo DEVS=$DEVS
+}
+
```

Comment:
I feel current configuration makes lot of sense and for future configuration, caller knows what modification he/she has done to docker-storage-setup file.

As a user I think what makes sense to me, is what's current storage driver, what's current lvm thin pool and which volume group it was carved out from and what disks were added.

What's future can easily be parsed from /etc/sysconfig/docker-storage-setup anyway.